Generate_Synt_Data_Set.ipynb

```diff
@@ -0,0 +1,580 @@
+{
```

Comment:
I suggest not putting jupyter notebooks in the root of the repository. If you need to put a notebook in this repo, I suggest to create a folder called `notebooks` and place it there.